docs/DeveloperGuide.adoc

```diff
@@ -775,6 +887,15 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Should be portable and can be ran from flash drives.
+.  Should be able to keep contact information secure.
+.  English should be the only language in the application.
+.  Application should not take more than 50mb of storage.
+.  A new user should be able to use the app with ease after 3 days of using it.
+.  AddressBook should continue to work offline or online.
+.  Should not use more than 100MB of link:#RAM[RAM].
+.  Should be able to work offline
```

Comment:
isn't this duplicate of `AddressBook should continue to work offline or online`?